fastlane/Fastfile

```diff
@@ -145,7 +146,20 @@ def distribute_builds(
       suffix: 'full',
       extension: 'nupkg',
       name: 'Windows Update'
-    }
+    },
+    windows_appx: {
+      binary_path: File.join(BUILDS_FOLDER, 'Studio-appx-signed', "Studio #{normalized_version}.0.appx"),
+      filename_core: 'win32',
+      extension: 'appx',
+      name: 'Windows (Appx)'
+    },
```

Comment:
I think we don't need the signed version of the appx.

Reply:
Yeah, we don't need ZIP, universal, etc. Let's leave it as is for completeness.